func_adl_xAOD/cms/aod/cms_functions.py

```diff
@@ -0,0 +1,41 @@
+# Some math utilities
```

Comment:
I guess this has slipped in by mistake?

Reply:
Ops, nice catch, yes! I'll kill that now!